pyproject.toml

```diff
@@ -75,6 +75,8 @@ httpx = "^0.27.0"
 pydantic = "^2.8.2"
 pyhumps = "^3.8.0"
 croniter = "^5.0.0"
+jsonlines = "^4.0.0"
+datetime = "5.5"
```

Comment:
datetime is a built-in lib, afaik? Not sure why it's listed/added here

Reply:
oh okay let me remove this